spec/requests/providers/property_value_spec.rb

```diff
@@ -4,83 +4,129 @@
   let(:legal_aid_application) { create :legal_aid_application, :with_applicant }
   let(:secure_id) { legal_aid_application.generate_secure_id }
 
-  describe 'provider property value test' do
-    describe 'GET /providers/applications/:id/property_value' do
-      before { get providers_legal_aid_application_property_value_path(legal_aid_application) }
+  describe 'GET /providers/applications/:id/property_value' do
+    before { get providers_legal_aid_application_property_value_path(legal_aid_application) }
 
-      it 'returns http success' do
-        expect(response).to have_http_status(:ok)
-        expect(response.body).to include(CGI.escapeHTML("How much is your client's home worth?"))
-      end
+    it 'returns http success' do
+      expect(response).to have_http_status(:ok)
+      expect(response.body).to include(CGI.escapeHTML("How much is your client's home worth?"))
```

Comment:
The patten used elsewhere in the code to handle HTML rendering of some characters is:
```ruby
expect(unescaped_response_body).to include("How much is your client's home worth?")
```
See `spec/support/request_helpers.rb`